client/state/data-layer/wpcom/sites/atomic/transfers/index.js

```diff
@@ -0,0 +1,46 @@
+import { translate } from 'i18n-calypso';
+import { ATOMIC_PLUGIN_INSTALL_INITIATE_WITH_TRANSFER } from 'calypso/state/action-types';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { registerHandlers } from 'calypso/state/data-layer/handler-registry';
+import { http } from 'calypso/state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'calypso/state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'calypso/state/notices/actions';
+
+const initiateAtomicTransferandInstall = ( action ) =>
+	http(
+		{
+			apiNamespace: 'wpcom/v2',
+			method: 'POST',
+			path: `/sites/${ action.siteId }/atomic/transfers/`,
+			body: {
+				software_set: action.softwareSet,
```

Comment:
This parameter should be optional, we're going to need to add plugin, theme, plugin file, theme file, and later options along with it.

Reply:
Made this optional in the action so just needs to account for that here.

Reply:
@lsl Attempted this here: https://github.com/Automattic/wp-calypso/pull/58709/commits/06e644cb1e2702e5381e0f32cf76f1fb6db1a192